raiden/raiden_service.py

```diff
@@ -947,7 +947,7 @@ def message_directtransfer(self, message):
 
         channel = graph.partneraddress_channel[message.sender]
 
-        if not channel.isopen:
+        if not channel.can_transfer:
```

Comment:
I think this is a bug, a node must accept a transfer if it cant't transfer but the channel is open. A regression test would be cool :)